### PR TITLE
Pennotool testcases

### PR DESCRIPTION
**Pull request by mhvis**
Done:
* Refactor for tests
* `read_exc` testcase

Todo:
* `get_qrekening`
* `get_sepa`

### CODE REVIEW

**Comment by mhvis** [comment]
Separated the workbook code from the process code in order to be able to test the Q-rekening code without having to create workbooks. That's why a lot of the function bodies are changed, sorry about that.

Couple questions:

* I added `strip` which removes the 4 spaces at the front of the descriptions. Is that okay?
* I changed `get_debet/credit_amounts` to have currency formatting with 2 decimal places. Is that okay?
* I don't know whether it still follows the requirements from user perspective, as I don't know the requirements, so you might wanna check that once more :laughing: .
* I changed `madaatdatum` in `sepa_headers` to `mandaatdatum`. Is that okay?

Still to do: fix floating point error so that test case succeeds

File: pennotools/qrekening/process.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+            'bedrag': amount,
+            'naam': p.name,
+            'beschrijving': description() if callable(description) else description,
+            'endtoendid': '{}{}'.format(p.id, date.today().strftime('%m%y')),
+        }
+
+    total = p.get_total()
+    while total > split:
+        rows.append(get_row(amount=split))
+        total -= 100
```
[the remark this answers is not in the document]
Comment (mhvis) [reply pennotools/qrekening/process.py:127]: :rofl: thanks

File: pennotools/qrekening/wb.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                # Ending person
+                current_person = None
+            else:
+                # Person boekstuk row
+                current_person.add_boekstuk(
+                    debet,
+                    row['Openstaand'],
+                    xldate_as_datetime(row['Fac/Bet Datum'], 0),  # 1900-based datemode
+                    row['Omschrijving'].strip(),
+                    # Todo: Wessel ik heb deze strip toegevoegd voor de 4 spaties voorop, is dat okay?
```
[the remark this answers is not in the document]
Comment (mhvis) [reply pennotools/qrekening/wb.py:54]: Wat jij wilt.

File: pennotools/qrekening/process.py [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+        'Deb Omschrijving': p.get_debet_description(),
+        'Deb Datum': p.get_debet_dates(),
+        'Deb Bedrag': p.get_debet_amounts(),
+        'Deb Open': p.get_debet_amounts(),
+        'Cred Tot Open': p.get_credit_total(),
+        'Cred Omschrijving': p.get_credit_description(),
+        'Cred Datum': p.get_credit_dates(),
+        'Cred Bedrag': p.get_credit_amounts(),
+        'Cred Open': p.get_credit_amounts(),
+        'Totaal Open Tekst': '{0:.2f}'.format(p.get_total()),
```
Comment (gwvdh) [inline pennotools/qrekening/process.py:60]:
```suggestion
        'Totaal Open Tekst': str(p.get_total(), Decimal(0.00)),
```

**Pull request merged**